rudder-core/src/main/resources/hooks.d/readme.adoc

```diff
@@ -7,7 +7,8 @@ execution.
 By default, a hook directory contains a template example and a Readme.txt file
 explaining the generalities about hooks and the specificities of that hook (parameters, etc).
 
-*Hooks must be executable*. All executable file will be used as hooks, and
+*Hooks must be executable*. All executable file will be used as hooks SAFE if
+it ends with one of the extensions listed in the file *non-exec-extensions.txt*, and
```

Comment:
I think it's worth explicitly mentioning .disabled here, to make it easy for users to, well, disable, hooks.